scripts/ez_color.pl

```diff
@@ -0,0 +1,104 @@
+use strict;
+use vars qw($VERSION %IRSSI);
+use Irssi;
+
+$VERSION = '1.0';
+
+%IRSSI = (
+	authors			=> "stryk",
+	contact			=> '$IRSSI{authors} + <Shift-2> + protonmail + <rhymes-with-pot> + <2-char Country Code for Switzerland>',
```

Comment:
$IRSSI{authors} is no defined at this time
and
this string will not interpolated